backend/User/validators.py

```diff
@@ -2,16 +2,32 @@
 import os
 
 def validate_image(file):
-	valid_mime_types = ['image/jpeg', 'image/png', 'image/gif']
-	valid_file_extensions = ['.jpg', '.jpeg', '.png', '.gif']
-	max_file_size = 2097152
+    """
```

Comment:
mmeh, as said before, summariesa are quite unnecessary and get easily obsolete if not kept up to date...